modules/consensus/consensusset.go

```diff
@@ -174,9 +175,13 @@ func (cs *ConsensusSet) ChildTarget(id types.BlockID) (target types.Target, exis
 
 // Close safely closes the block database.
 func (cs *ConsensusSet) Close() error {
+	// TODO: is this lock necessary?
 	cs.mu.Lock()
 	defer cs.mu.Unlock()
```

Comment:
furthermore, `Close` is exported, so people will be assuming that there's a lock. That doesn't necessarily mean that it needs a lock, but if it's a larger function it should have one so that devs don't make the wrong assumptions.